contributor/contributors-guide.md

```diff
@@ -6,6 +6,41 @@ permalink: /contributors-guide/
 **Please read the [How to Contribute] page first before reading this guide.** This page contains additional project
 documentation.
 
+Accumulo previously used [JIRA], but now uses GitHub issues.
+All new issues should be opened using GitHub. If working an existing JIRA
+issue, please do the following :
+
+ * Open a new GitHub issue or pull request.
+ * Link the GitHub issue to the JIRA issue.
+ * Link the JIRA issue to the GitHub issue.
```

Comment:
This can be done simultaneously by simply adding a link to the JIRA issue in the GitHub issue/PR body (or in a comment). An ASF bot will automatically comment on the JIRA issue adding a link.

Reply:
I am going to just leave this as is since this will be deleted eventually.